### PR TITLE
{vis}[foss/2019b] ParaView v5.6.2 w/ Python 3.7.4

### PR DESCRIPTION
**Pull request by furstj**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by furstj** [comment]
There is ParaView 5.7.0 but its build system has been re-designed (see https://blog.kitware.com/paraview-5-7-0-release-notes/). Moreover, the OpenFOAM-7 as well as OpenFOAM-v1906 are still distributed with ParaView-5.6.

**Comment by boegel** [comment]
@furstj Any idea how much effort it would be to (also) have an easyconfig for ParaView 5.7.0?

File: easybuild/easyconfigs/p/ParaView/ParaView-5.6.2-foss-2019b-Python-3.7.4-mpi.eb [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+configopts += "-DVTK_USE_SYSTEM_NETCDF=ON "
+
+# OpenGL & Mesa
+configopts += '-DOPENGL_glu_LIBRARY=$EBROOTLIBGLU/lib/libGLU.%s ' % SHLIB_EXT
+configopts += '-DOPENGL_INCLUDE_DIR=$EBROOTMESA/include -DOPENGL_gl_LIBRARY=$EBROOTMESA/lib/libGL.%s ' % SHLIB_EXT
+configopts += '-DVTK_OPENGL_HAS_OSMESA=ON '
+configopts += '-DOSMESA_INCLUDE_DIR=$EBROOTMESA/include -DOSMESA_LIBRARY=$EBROOTMESA/lib/libOSMesa.%s ' % SHLIB_EXT
+
+
+# Without internet connection turn off testing (uncomment the following line)
```
Comment (smoors) [inline easybuild/easyconfigs/p/ParaView/ParaView-5.6.2-foss-2019b-Python-3.7.4-mpi.eb]: the following line is un-commented already, so please adapt the comment accordingly?

File: easybuild/easyconfigs/p/ParaView/ParaView-5.6.2-foss-2019b-Python-3.7.4-mpi.eb [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+configopts += '-DPARAVIEW_ENABLE_FFMPEG=ON '
+configopts += '-DPARAVIEW_ENABLE_PYTHON=ON '
+
+# Python
+configopts += "-DVTK_WRAP_PYTHON=ON "
+configopts += "-DVTK_PYTHON_VERSION=%(pyver)s "
+configopts += "-DPYTHON_INCLUDE_DIR=$EBROOTPYTHON/include/python%(pyshortver)sm "
+configopts += "-DPYTHON_LIBRARY=$EBROOTPYTHON/lib/libpython%%(pyshortver)sm.%s " % SHLIB_EXT
+#
+# configopts += '-DPYTHON_EXECUTABLE=$EBROOTPYTHON/bin/python3.7m '
```
Comment (smoors) [inline easybuild/easyconfigs/p/ParaView/ParaView-5.6.2-foss-2019b-Python-3.7.4-mpi.eb]: remove this commented line?

File: easybuild/easyconfigs/p/ParaView/ParaView-5.6.2-foss-2019b-Python-3.7.4-mpi.eb [hunk 3]
```diff
@@ -0,0 +1,82 @@
+easyblock = 'CMakeMake'
+
+name = 'ParaView'
+version = '5.6.2'
+versionsuffix = '-Python-3.7.4-mpi'
```
[the remark this answers is not in the document]
Comment (furstj) [reply easybuild/easyconfigs/p/ParaView/ParaView-5.6.2-foss-2019b-Python-3.7.4-mpi.eb]: Corrected in latest commit

File: easybuild/easyconfigs/p/ParaView/ParaView-5.6.2-foss-2019b-Python-3.7.4-mpi.eb (hunk 1 above)
Comment (furstj) [reply easybuild/easyconfigs/p/ParaView/ParaView-5.6.2-foss-2019b-Python-3.7.4-mpi.eb]: The line was commented out

File: easybuild/easyconfigs/p/ParaView/ParaView-5.6.2-foss-2019b-Python-3.7.4-mpi.eb (hunk 2 above)
Comment (furstj) [reply easybuild/easyconfigs/p/ParaView/ParaView-5.6.2-foss-2019b-Python-3.7.4-mpi.eb]: The line was removed

**Comment by furstj** [comment]
@boegel I don't know how difficult will be to prepare ParaView 5.7.0. I did some work on it. It compiles, but the resulting executable does not work yet. I expect to spend more time on it during this winter...

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 11 out of 11 (1 easyconfigs in this PR)
node371.hydra.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/e41a865eceff7d4d615f57c7d96cefeb for a full test report.

**Comment by smoors** [comment]
Going in, thanks @furstj!

**Pull request merged**